src/Struct/FindologicProduct.php

```diff
@@ -456,7 +456,9 @@ protected function setAttributeProperties(): void
 
     protected function setAdditionalAttributes(): void
     {
-        $this->attributes[] = new Attribute('shipping_free', [$this->product->getShippingFree() ? 1 : 0]);
+        $value = $this->product->getShippingFree() ? "yes" : "no";
```

Comment:
`$value` => `$translationKey` and make it so it contains the entire translation key.